CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 8.0.7
+### General
```

Comment:
The last bullet point isn't needed, since that is an implementation detail.
Also, perhaps we should reword it to something like this?
I also added a specific note for people who were using a workaround for the Platform class issue like:

```
if (kIsWeb) {
  FilePickerWeb()
} else {
  FilePicker()
}
```

```suggestion
### General
- Fixes an issue relating to incorrect registration of platform-specific implementations. [#1555](https://github.com/miguelpruivo/flutter_file_picker/issues/1555).
- Fixes an issue that required users to use `kIsWeb` and `FilePickerWeb` when on the web.
```

Reply:
Fully agree here, thanks!